### PR TITLE
Upload Feature Branch!

### PR DESCRIPTION
**Pull request by tihuan**
### Description

Live link: https://upload-frontend.dev.genepi.czi.technology/data/samples

GIF:
demo

#### Issue
Epic: https://app.clubhouse.io/genepi/epic/119804/v3-aspen-user-genome-fasta-file-upload?ct_workflow=all

### Test plan

Write how your changes are tested, or give a convincing reason why they can't be tested automatically.

### CODE REVIEW

File: src/frontend/pages/upload/[[...params]].tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
```
Comment (phoenixAja) [inline src/frontend/pages/upload/[[...params]].tsx:1]: is this file supposed to be named this? lol it looks funny

Comment (tihuan) [reply src/frontend/pages/upload/[[...params]].tsx:1]: Ahaha yeah Next.js came up with this interesting idea lol

https://nextjs.org/docs/routing/dynamic-routes

Comment (phoenixAja) [reply src/frontend/pages/upload/[[...params]].tsx:1]: oh very interesting

File: src/frontend/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "local-dev": "API_URL=http://backend.genepinet.local:3000 npm run dev --  -p 8000",
     "tests": "jest --config jest/config.js",
     "e2e": "DEBUG=pw:api jest --config jest/e2e.config.js",
-    "e2e-staging-from-local": "AWS_PROFILE=genepi-dev FRONTEND_URL=https://staging.genepi.czi.technology/ DEPLOYMENT_STAGE=staging make e2e",
+    "e2e-staging-from-local": "AWS_PROFILE=genepi-dev FRONTEND_URL=https://staging.genepi.czi.technology DEPLOYMENT_STAGE=staging make e2e",
```
Comment (shanaxel42) [inline src/frontend/package.json:14]: Was this change intended?

Comment (tihuan) [reply src/frontend/package.json:14]: Ah yes that's needed and intended, otherwise URL might get double forward slashes!

**Review by shanaxel42 — APPROVED** [review]
